### DISCUSSION TITLE
posts/paper-reading/cheating-llm-bench/

### DISCUSSION THREAD

**Discussion by giscus** [huskydoge/comments-set#3] [Announcements]
# posts/paper-reading/cheating-llm-bench/

TL;DR: TODO

    We basically do three types of research: Y, F, X; Y = F(X). 80% focus on F, optimizing models; 15% focus on X, presenting data; 5% focus on Y, creating new meaningful tasks.— First Taught by a Mentor of Mine

There are many well-known LLM benchmarks, such as AlpacaEval 2.0, Arena-Hard-Auto, and MTBench.

http://localhost:58831/posts/paper-reading/cheating-llm-bench/

**Comment by Kr-Panghu** [comment]
Really a fantastic job, thanks Benhao.
